runtests.py

```diff
@@ -19,7 +19,7 @@ def runtest_exists(dir):
     return os.path.exists(f"{dir}/runtest.sh")
 
 def run():
-    cpu_count = multiprocessing.cpu_count()
+    cpu_count = max(multiprocessing.cpu_count() - 2, 1)
```

Comment:
whats the context behind this?

Reply:
my laptop was basically useless while the tests ran because no cpu cores for anything else lol. This frees up 2 cores for some usability of other functionality (eg. vscode, browser, etc)